src/main/resources/assets/appliedenergistics2/lang/en_US.lang

```diff
@@ -184,6 +184,8 @@ gui.appliedenergistics2.Lumen=Lumen
 gui.appliedenergistics2.Empty=Empty
 gui.appliedenergistics2.Stored=Stored
 gui.appliedenergistics2.Cancel=Cancel
+gui.appliedenergistics2.ETA=ETA:
+gui.appliedenergistics2.ETAFormat=HH:mm:ss
```

Comment:
Is it possible to give translators only one String? (so on need, they can switch orders or whatever)